.github/CODEOWNERS.md

```diff
@@ -0,0 +1,8 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @cuappdev/podcast-ios will be requested for
+# review when someone opens a pull request.
+*       @cuappdev/podcast-ios
```

Comment:
Do we need to add team members to this @cuappdev/podcast-ios or does it just alert anyone watching the repo?

Reply:
Yes @cuappdev/podcast-ios would be a team just like @cuappdev/developers. As I watch this repo you wouldn't want me to be able to review and approve your pull requests.

Reply:
Awesome I added @cuappdev/podcast-ios

Reply:
Feel free to merge

Reply:
Cool cool. See how it works for your pod and then maybe others will adapt it too.